src/transforms/transform-deployment.ts

```diff
@@ -101,7 +100,7 @@ async function getCommitMessagesSinceLastSuccessfulDeployment(
 // https://developer.atlassian.com/cloud/jira/software/rest/api-group-builds/#api-deployments-0-1-bulk-post
 // Deployment state - Jira: Can be one of unknown, pending, in_progress, cancelled, failed, rolled_back, successful
 function mapState(state: string): string {
-	switch (state) {
+	switch (state.toLowerCase()) {
```

Comment:
Is it happening? Is there a guarantee that `state` is always defined/not null?

Reply:
unsure about always defined, we could `state?.toLowerCase()` to protect that case.